### PR TITLE
Implement 7 Tests & Methods

### LINKED ISSUE #2: Implement 7 Tests & Methods
BDD ID createVar(const std::string &label)

const BDD ID &True()

const BDD ID &False()

bool isConstant(const BDD ID f )

bool isVariable(const BDD ID x)

BDD ID topVar(const BDD ID f )

size t uniqueTableSize()

### PR DESCRIPTION
**Pull request by Giorgi-Solo**
**I have implemented tests and methods for the first 6 functions**
BDD ID createVar(const std::string &label)
const BDD ID &True()
const BDD ID &False()
bool isConstant(const BDD ID f )
bool isVariable(const BDD ID x)
BDD ID topVar(const BDD ID f )

At some point, I needed to get size of uniqueTable, so I implemented size t uniqueTableSize() method as well

### CODE REVIEW

**Comment by Giorgi-Solo** [comment]
For this, we should use merge pull, not squash. Because all commits from the branch should be added to the master

File: src/Manager.cpp [hunk 1]
```diff
@@ -11,23 +11,59 @@ Manager::Manager()
...
+* Adds new node to UniqueTable
+* @param new node label
+* @return id of new node
+*/
 BDD_ID Manager::createVar(const std::string &label)
-{}
+{
+    BDD_ID newVarId = uniqueTable.size();
+    BDD_ID i;
+    for(i = 0; i < newVarId; ++i)          // to check if we already have that variable
```
Comment (Erhan-Laurentiu) [inline src/Manager.cpp:24]: I think for aesthetics reasons you can put the definition for "i" inside the for. (ex for(BDD_ID i = 0; .....)

File: src/Manager.h [hunk 2]
```diff
@@ -9,6 +9,8 @@
 #include <iostream>
 #include <vector>
 
+using namespace std;
```
Comment (Erhan-Laurentiu) [inline src/Manager.h:12]: If you added this probably you can remove the "std::" from "std::string"/ "std::vector"(from this file). Also you could add it to the .cpp as well and remove the "std::" from there as well

File: src/Manager.cpp (hunk 1 above)
Comment (Giorgi-Solo) [reply src/Manager.cpp:24]: Yes. But I am not sure if definition in for() works for all compilers.

Comment (Erhan-Laurentiu) [reply src/Manager.cpp:24]: This can be leaved like it is then

File: src/Manager.h (hunk 2 above)
Comment (Giorgi-Solo) [reply src/Manager.h:12]: ok

**Pull request merged**